### DISCUSSION TITLE
Defining the "Dimension" of a parameter in Mobi

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1698] [Questions & Problems] [answered]
Dear all，
I want to define a "dimension" in mobi, the unit should be "Inversed mass (/g)''. It seems that there is no such "dimension". I also tried to use "dimensionless" to represent it, but it doesn't seem to fit my purpose.
What should I do?

**Comment by Yuri05** [comment, marked as answer]
You can extend the MoBi dimension factory ("**C:\ProgramData\Open Systems Pharmacology\MoBi\11.2\OSPSuite.Dimensions.xml**") with the definition of new _Inversed mass_ dimension and restart MoBi

```
    <Dimension name="Inversed mass" baseUnit="1/kg" defaultUnit="1/g">
      <BaseRepresentation lengthExponent="0" massExponent="-1" timeExponent="0" electricCurrentExponent="0" temperatureExponent="0" amountExponent="0" luminousIntensityExponent="0" />
      <Units>
        <Unit name="1/kg" factor="1" />
        <Unit name="1/g" factor="1E3" />
        <Unit name="1/mg" factor="1E6" />
        <Unit name="1/µg" factor="1E9" />
        <Unit name="1/ng" factor="1E12" />
        <Unit name="1/pg" factor="1E15" />
      </Units>
    </Dimension>
```

**Reply by SCZ1174** [reply]
Thank you so much! It worked.

**Comment by PavelBal** [comment]
@Yuri05 Didn't we think about a FAQ section somewhere? This would be a good topic.
